src/utils/rate-limit.js

```diff
@@ -59,9 +59,9 @@ export function throttle(win, callback, minInterval) {
  * invoked.
  *
  * @param {!Window} win
- * @param {function()} callback
+ * @param {function(...*)} callback
  * @param {number} minInterval the minimum time interval in millisecond
- * @returns {function()}
+ * @returns {function(...*)}
```

Comment:
Is this the right signature for a function that just forwards its arguments? I made this change because `check-types` showed a type mismatch for the `input-debounced` debounce callback which is `{function(!Event)}`, not `{function()}`